client/src/pages/index.tsx

```diff
@@ -54,151 +68,127 @@ const IndexPage = ({location}: IndexPageProps) => {
   return (
     <Layout location={location}>
 
-      <J40MainGridContainer fullWidth={true}>
-        <AlertWrapper />
-      </J40MainGridContainer>
+      <J40MainGridContainer className={'j40-about-page'}>
```

Comment:
I'm guessing something like this might allow us to add the headers from the design?

```jsx
<J40MainGridContainer className={'j40-about-page'}>

   // Seems as though we can use the padding token from USWDS directly in-line 
   // (kinda like tailwind CSS!). After we get our h1 - hN defined from design 
   // we may not have to overide them like this. Just one of a hundred ways
   // to implement this headings

    <h2 className={'padding-top-3'}>How to get started</h2>
    <AboutCardsContainer>
    :
    :
 ```
 
 Might need to also reduce the about-card's top margin:
 ```css
   .j40-aboutcard-card {
    margin: 3rem 0 3rem 0;
 ```

Reply:
Good call.